.github/workflows/service_test_hdfs.yml

```diff
@@ -96,6 +98,8 @@ jobs:
             -e WEBHDFS_CONF_dfs_webhdfs_enabled=true \
             -e CORE_CONF_hadoop_http_staticuser_user=root \
             -e HDFS_CONF_dfs_permissions_enabled=false \
+            -e HDFS_CONF_dfs_support_append=true \
+            -e HDFS_CONF_dfs_replication=1 \
```

Comment:
You are right. We need to configure `dfs_replication` correct.

We can add a `hdfs-site.xml` file in `fixtures/hdfs` and copy into `/home/runner/hadoop-3.2.4/etc/hadoop` in CI.